lib/cdo/github.rb

```diff
@@ -39,6 +39,26 @@ def self.merge_pull_request(pr_number)
     response['merged']
   end
 
+  # Creates and merges a pull request.
+  # @param base [String] The base branch of the requested pull request.
+  # @param head [String] The head branch of the requested pull request.
+  # @param title [String] The title of the requested pull request.
+  # @return [nil | Integer] The PR number of the newly created DTT if successful
+  #   or nil if unsuccessful.
+  def self.create_and_merge_pull_request(base:, head:, title:)
```

Comment:
Given that this is a common paradigm (also to be used in `bin/dotd`), seems worth bottling into a single method call.